test/src/com/xilinx/rapidwright/edif/TestEDIFParser.java

```diff
@@ -123,8 +124,14 @@ public void loadEDIFSingleThreaded() throws IOException {
 
     }
 
-    @Test
-    public void testGZIPEDIFParsing(@TempDir Path tempDir) throws IOException {
+    @ParameterizedTest
+    @ValueSource(booleans = { false, true })
+    public void testGZIPEDIFParsing(boolean decompressToDisk, @TempDir Path tempDir)
+            throws IOException {
+        if (decompressToDisk) {
+            System.setProperty("RW_DECOMPRESS_GZIPPED_EDIF", "1");
```

Comment:
I'm torn about using a String to reference the parameter here (and below in the sister test).  Usually this is bad practice, but I think the name of the boolean parameter should probably be the same as the name of the interface parameter.  We could add a separate string parameter `String RW_DECOMPRESS_GZIPPED_EDIF_NAME` for example, and reference that, but it doesn't seem the best.  Another option might be to create a named Boolean class (String+boolean), but that seems like overkill. Ideally, we would never change the name of a paramter (just create a new one), but never say never, I guess.